tools/handle_ocp_versions.py

```diff
@@ -26,6 +26,7 @@ def handle_arguments():
     parser.add_argument('--name-override', type=str)
     parser.add_argument('--version-override', type=str)
     parser.add_argument('--skip-verify', action="store_true")
+    parser.add_argument('--single-version-only', type=str, default='false')
```

Comment:
Any reason not to use `action='store_true'` here? If the flag is there, it's True. If it's not, it's False.

```
[nav] In [7]: import argparse
         ...: parser = argparse.ArgumentParser()
         ...: parser.add_argument('--foo', action='store_true')
         ...: print(parser.parse_args([]))
         ...: print(parser.parse_args(['--foo']))
Namespace(foo=False)
Namespace(foo=True)
```

Reply:
Well, it could be used. But we now use it as follows:
```
--single-version-only=${SINGLE_VERSION_ONLY}
```
So it's simpler to pass it.

Reply:
It could be used, but we now use it as follows:

```
--single-version-only=${SINGLE_VERSION_ONLY}
```

So it's more simple to use it as is from bash.

Reply:
Can resolve conversation